plugins/modules/web_infrastructure/jira.py

```diff
@@ -314,7 +314,14 @@ def request(url, user, passwd, timeout, data=None, method=None):
     if info['status'] not in (200, 201, 204):
         error = json.loads(info['body'])
         if error:
-            module.fail_json(msg=error['errorMessages'])
+            msg = ''
+            for key in ('errorMessages', 'errors'):
+                if error.get(key):
+                    msg += str(error[key])
+            if msg:
+                module.fail_json(msg=msg)
+            else:
+                module.fail_json(msg=error)
         else:
             # Fallback print body, if it cant be decoded
             module.fail_json(msg=info['body'])
```

Comment:
```suggestion
            msg = []
            for key in ('errorMessages', 'errors'):
                if error.get(key):
                    msg.append(error[key])
            if msg:
                module.fail_json(msg=to_native(', '.join(msg)))
            else:
                module.fail_json(msg=to_native(error))
        else:
            # Fallback print body, if it cant be decoded
            module.fail_json(msg=to_native(info['body']))
```
1. hey, in case of 2 error[key] concatenated by `msg += `, can't we get something like 'error1error2' (without any separators)?
2. there is the `to_native` function from module_utils availeble  for such cases (provides proper string handling independently from python version)